src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -364,11 +365,19 @@
 (defn- community-card-page-view
   [id]
   (let [{:keys [id joined name images]
-         :as   community} (rf/sub [:communities/community id])]
-    (when community
-      (when joined
-        (rf/dispatch [:activity-center.notifications/dismiss-community-overview id]))
-      [community-scroll-page id joined name images])))
+         :as   community} (rf/sub [:communities/community id])
+        fetching?         (rf/sub [:communities/fetching-community id])]
+    (if community
+      (do
+        (when joined
+          (rf/dispatch [:activity-center.notifications/dismiss-community-overview id]))
+        [community-scroll-page id joined name images])
+      [rn/view
+       {:style style/tmp-fetching}
+       [not-implemented/not-implemented
+        (if (fetching?)
+          [rn/text "Fetching community..."]
```

Comment:
Can we add `:accessibility-id` here, so qa team can create e2e tests? 
(Its just a placeholder but we can keep those ids in new design.)